test/Bridges/lazy_bridge_optimizer.jl

```diff
@@ -14,7 +14,7 @@ MOIU.@model(NoRSOCModel,
             (MOI.EqualTo, MOI.GreaterThan, MOI.LessThan),
             (MOI.Zeros, MOI.Nonnegatives, MOI.Nonpositives, MOI.SecondOrderCone,
              MOI.ExponentialCone, MOI.PositiveSemidefiniteConeTriangle),
-            (),
+            (MOI.PowerCone),
```

Comment:
This is possibly the source of the error because `(MOI.PowerCone)` isn't a tuple. Try `(MOI.PowerCone,)` instead.